### PR TITLE
Updated docs

### PR DESCRIPTION
**Pull request by Narcos101**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -78,6 +79,14 @@ Some quick links to useful parts of the documentation follow below for your conv
 
 To get started using rotki, check out the [detailed usage guide](https://rotki.readthedocs.io/en/latest/usage_guide.html).
 
+## Application Preview
+
+<div style="display: flex; justify-content: space-between">
```
Comment (LefterisJP) [inline README.md:84]: This looks really small currently @Narcos101. You can see it in your own fork. Can you make both pics take the max size and be the same size? https://github.com/Narcos101/rotki/tree/develop#application-preview

File: .github/profile/README.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
+<p align="center">
```
Comment (LefterisJP) [inline .github/profile/README.md:1]: What exactly is this file? We already have README.md. What is this supposed to be?

**Review by LefterisJP — COMMENTED** [review]
Also please try to keep commits to a minimum. Each commit doing a specific full change. For example adding profile readme and editing should be one commit.

File: .github/profile/README.md (hunk 2 above)
Comment (kelsos) [reply .github/profile/README.md:1]: Is this to have an organization [profile](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/managing-your-profile-readme)?

Comment (Narcos101) [reply .github/profile/README.md:1]: Yes, The readme is for the organization profile. for example:- https://github.com/github.

**Review by LefterisJP — COMMENTED** [review]
hey @Narcos101 the images still look weird:



Also can you do interactive rebase and keep this to 2 commits instead of 4 as asked in the last review?

**Comment by Narcos101** [comment]
> hey @Narcos101 the images still look weird:
> 
> 
> 
I had aligned them perfectly both the times, and it was perfectly shown while editing and previewing it. Once I commit it, it breaks entirely.

**Review by LefterisJP — COMMENTED** [review]
This seems to still show incorrectly and has some kind of weird watermark on the 1st picture

**Pull request closed without merging**